articles/connections/passwordless/configure-passwordless-connection.md

```diff
@@ -0,0 +1,137 @@
+
+![Enable SMS Passwordless](/media/articles/connections/passwordless/connections-passwordless-list.png)
+
+2. Enter your **Twilio Account SID** and **Twilio Auth Token**.
+
+::: note
+To learn how to find your Twilio SID and Auth Token, see the Twilio docs on the[Application SID](https://support.twilio.com/hc/en-us/articles/223136607-What-is-an-Application-SID-) and the [AuthToken](https://support.twilio.com/hc/en-us/articles/223136027-Auth-Tokens-and-How-to-Change-Them).
+:::
+
+![Configure SMS Passwordless](/media/articles/connections/passwordless/connections-passwordless-sms.png)
+
+3. Select your **SMS Source** and, depending on your selection, enter either your **Twilio Messaging Service SID** or a **From** phone number. Users will see what you enter as the sender of the SMS.
+
+::: note
+To learn about using Twilio Copilot, see [Sending Messages with Copilot](https://www.twilio.com/docs/api/rest/sending-messages-copilot).
+:::
+
+4. In **Message**, enter the body text of the SMS.
+
+**Make sure that your Message includes `@@password@@` since this is the placeholder that will be replaced with the one-time password that is sent to the user.**
+
+You can use multiple languages in your SMS Messages. You can choose the language used by calling the [`passwordless/start`](/docs/api/authentication/reference#get-code-or-link) endpoint and setting the `x-request-language` header to the language of your choice. Otherwise, the language used will be extracted from the `accept-language` header (which is automatically set by your browser).
+
+The **Message** area accepts Markdown and Liquid syntax. If you choose to use Liquid, you can programmatically construct elements of the message. For example, you can reference the `request_language` parameter to change the language of the message:
+
+```text
+{% if request_language contains 'dutch' %}
+ Hier is uw verificatie code: @@password@@
+{% endif %}
+
+{% if request_language contains 'fr-FR' %}
+ Ceci est votre code: @@password@@
+{% endif %}
+```
+
+The following parameters are available to you when defining the message template (regardless of which syntax you use):
+
+| Parameter | Description |
+| - | - |
+| `password` or `code` | The one-time password to send to the user |
+| `phone_number ` | The user's phone number |
+| `application.name` | The name of the application to which the user is logging in |
+| `request_language` | The requested language to be used for the SMS message's content |
+
+5. Adjust settings for your **OTP Expiry** and **OTP Length**.
+
+| Parameter | Description |
+| - | - |
+| OTP Expiry | The amount of time before the one-time password expires |
+| OTP Length | The length of the one-time password sent to the user |
+
+If you choose to extend the OPT expiry time, we recommend increasing the OTP length for security reasons. Otherwise, the malicious party has a longer time frame to try to guess a short code.
```

Comment:
What is a short code? We haven't used this vocabulary anywhere else.

malicious parties have